src/Camera.cpp

```diff
@@ -58,3 +58,36 @@ void Camera::RotateAround(float deltaX, float deltaY)
 }
 
 
+void Camera::Zoom(bool in)
+{
+	const float zoomSpeed = 0.2;
+
+	glm::vec3 direction = zoomSpeed*(normalize(forward-pos));
+	if(!in){
```

Comment:
Left bracket should go to new line to keep consistent with the rest of the code

Reply:
d